PWGCF/FemtoDream/femtoDreamPairTaskTrackV0.cxx

```diff
@@ -0,0 +1,289 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file femtoDreamPairTaskTrackTrack.cxx
+/// \brief Tasks that reads the track tables used for the pairing and builds pairs of two tracks
+/// \author Andi Mathis, TU München, andreas.mathis@ph.tum.de
+
+#include "PWGCF/DataModel/FemtoDerived.h"
+#include "FemtoDreamParticleHisto.h"
+#include "FemtoDreamEventHisto.h"
+#include "FemtoDreamPairCleaner.h"
+#include "FemtoDreamContainer.h"
+#include "FemtoDreamDetaDphiStar.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/runDataProcessing.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/ASoAHelpers.h"
+
+using namespace o2;
+using namespace o2::analysis::femtoDream;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+namespace
+{
+static constexpr int nCuts = 4;
+static const std::vector<std::string> cutNames{"MaxPt", "PIDthr", "nSigmaTPC", "nSigmaTPCTOF"};
+static const float cutsArray[nCuts] = {4.05f, 0.75f, 3.f, 3.f};
+
+static constexpr int nNsigma = 3;
+static constexpr float kNsigma[nNsigma] = {3.5f, 3.f, 2.5f};
+
+enum kPIDselection {
+  k3d5sigma = 0,
+  k3sigma = 1,
+  k2d5sigma = 2
+};
+
+enum kDetector {
+  kTPC = 0,
+  kTPCTOF = 1,
+  kNdetectors = 2
+};
+} // namespace
+
+struct femtoDreamPairTaskTrackV0 {
+
+  /// Particle 1 (track)
+  Configurable<int> ConfPDGCodePartOne{"ConfPDGCodePartOne", 2212, "Particle 1 - PDG code"};
+  Configurable<int> ConfCutPartOne{"ConfCutPartOne", 5542986, "Particle 1 - Selection bit from cutCulator"};
+  Configurable<std::vector<int>> ConfPIDPartOne{"ConfPIDPartOne", std::vector<int>{2}, "Particle 1 - Read from cutCulator"}; // we also need the possibility to specify whether the bit is true/false ->std>>vector<std::pair<int, int>>int>>
+  Configurable<LabeledArray<float>> cfgCutArray{"cfgCutArray", {cutsArray, nCuts, cutNames}, "Particle selections"};
+  Configurable<int> cfgNspecies{"ccfgNspecies", 4, "Number of particle spieces with PID info"};
+
+  /// Partition for particle 1
+  Partition<aod::FemtoDreamParticles> partsOne = (aod::femtodreamparticle::partType == uint8_t(aod::femtodreamparticle::ParticleType::kTrack)) &&
```

Comment:
It will result in the same configurable-in-partition problem as previously, wouldn't my suggestion work?

Reply:
I just make the PR before reading your suggestion. It will be modified in the next commit, together with the track-track task. OK?